customization-base/README.md

```diff
@@ -25,10 +25,7 @@ customizations are supported:
 - [Add an annotation to a class](#add-an-annotation-to-a-class)
 - [Add an annotation to a method](#add-an-annotation-to-a-method)
 - [Remove an annotation from a class](#remove-an-annotation-from-a-class)
-- [Refactor: Rename a class](#refactor-rename-a-class)
 - [Refactor: Rename a method](#refactor-rename-a-method)
-- [Refactor: Generate the getter and setter methods for a property](#refactor-generate-the-getter-and-setter-methods-for-a-property)
-- [Refactor: Rename a property and its corresponding getter and setter methods](#refactor-rename-a-property-and-its-corresponding-getter-and-setter-methods)
```

Comment:
What are the alternatives for these removed features?

Reply:
Yeah... Usually the power of the language server is on the rename symbol part. I think it will help modify the code that refs the symbol.

java-parser does have a symbol-solver lib. But from the experience (used it 3 years ago to auto rename some API in mgmt lib), it is not very reliable.